backend/routes/image_generation/image_generation_api.js

```diff
@@ -2,13 +2,14 @@ import express from "express";
 import axios from "axios";
 import { v4 as uuidv4 } from "uuid";
 import { query, check, validationResult } from "express-validator";
-import { saveImageJobData, getImageJobData, updateImageJobStatus, connectToDB, doesJobIdExist } from "../database/dbService.js";
+import { saveImageJobData, getImageJobData, updateImageJobStatus, connectToDB, doesJobIdExist, cancelImageJob} from "../database/dbService.js";
 import { authenticate } from "../auth/authentication.js";
 
 const router = express.Router();
 const baseUrl = "https://stablehorde.net/api/v2";
 const apiKey = process.env.API_KEY;
-
+const maxErrorRetries = process.env.MAX_ERROR_RETRIES;
```

Comment:
max tries logic is supposed to be on front end

Reply:
Removed.